Monika After Story/game/script-brbs.rpy

```diff
@@ -278,22 +298,13 @@ label monika_idle_coding:
 
 label monika_idle_coding_callback:
     if mas_isMoniNormal(higher=True):
-        python:
-            coding_brb_start = mas_getEV("monika_idle_coding").last_seen
-            wb_quips = [
-                _("What else did you want to do today?"),
-                _("Is there anything else you wanted to do today?"),
-                _("What else should we do today?")
-            ]
-            wb_quip = renpy.random.choice(wb_quips)
-
-        if not mas_timePastSince(coding_brb_start, datetime.timedelta(minutes=20)):
-            m 1eub "Oh, done already, [player]?"
-
+        $ wb_quip = mas_brbs.get_wb_quip()
+        if mas_brbs.was_idle_for_at_least("monika_idle_coding", datetime.timedelta(minutes=20)):
```

Comment:
Shouldn't there be `if not` here? The 'done already' line is supposed to proc when the player is away for less than 20 minutes.